resourcecontrollerv2/resource_controller_v2_examples_test.go

```diff
@@ -77,7 +77,8 @@ var _ = Describe(`ResourceControllerV2 Examples Tests`, func() {
 		keyName                    string = "RcSdkKey1Go"
 		keyUpdateName              string = "RcSdkKeyUpdate1Go"
 		targetRegion               string = "global"
-		resourceGroupID            string = "testResourceGroupID"
+		// Replacing the placeholder value to the resourceGroupID
+		resourceGroupID string = "192ea4f81d444a4cbfc051353b6f8d2e"
```

Comment:
Variables that are sensitive or can change between tests should not be defined here. If you want to use something like that, you need to define it as an environment variable and load it. See [THIS](https://github.com/IBM/platform-services-go-sdk/blob/main/resourcecontrollerv2/resource_controller_v2_examples_test.go#L104) example.
Furthermore I see that the description of the `resourceGroup` and `resourceGroupID` variables are identical, so cannot the first one be used for this purpose and the second one removed?

Reply:
Reverted this change to the previous state for this variable resource_group_id